src/input/datetime.rs

```diff
@@ -575,7 +599,7 @@ impl TzInfo {
             }
             let offset_seconds: f64 = offset_delta.call_method0(intern!(py, "total_seconds"))?.extract()?;
             let offset = offset_seconds.round() as i32;
-            Ok(op.matches(self.seconds.cmp(&offset)).into_py(py))
+            op.matches(self.seconds.cmp(&offset)).into_py_any(py)
```

Comment:
Probably a lot more of the `.into_py(py)` calls could have been replaced with just `.into_py_any(py)` but I couldn't stop myself using more efficient alternatives where I spotted them 🙈